doc/guides/tfm/build.rst

```diff
@@ -118,3 +118,42 @@ following CMake snippet in your CMakeLists.txt file.
    The ``TFM_CMAKE_OPTIONS`` is a list so it is possible to append multiple
    options. Also CMake generator expressions are supported, such as
    ``$<1:-DFOO=bar>``
+
+Footprint and Memory Usage
+**************************
+
+The build system offers targets to view and analyse RAM and ROM usage in generated images.
+The tools run on the final images and give information about size of symbols and code being used in both RAM and ROM.
+For more information on these tools look here: :ref:`footprint_tools`
+
+Use the ``tfm_ram_report`` to get the RAM report for the secure firmware (tfm_s).
+
+.. zephyr-app-commands::
+    :tool: all
+    :app: samples/hello_world
+    :board: mps2_an521_ns
+    :goals: tfm_ram_report
+
+Use the ``tfm_rom_report`` to get the ROM report for the secure firmware (tfm_s).
+
+.. zephyr-app-commands::
+    :tool: all
+    :app: samples/hello_world
+    :board: mps2_an521_ns
+    :goals: tfm_rom_report
+
+Use the ``bl2_ram_report`` to get the RAM report for mcuboot, if enabled.
+
+.. zephyr-app-commands::
+    :tool: all
+    :app: samples/hello_world
+    :board: mps2_an521_ns
+    :goals: bl2_ram_report
+
+Use the ``bl2_rom_report`` to get the ROM report for mcuboot, if enabled.
```

Comment:
similar here, I think it is important to make it clear to the reader that this is not just regular MCUboot, but TF-M build MCUboot.
```suggestion
Use the ``bl2_rom_report`` to get the ROM report for TF-M enabled MCUboot.
```